config/global.ini.php

```diff
@@ -455,6 +455,12 @@
 ; Default is 0 (i.e., bust frames on the Settings forms).
 enable_framed_settings = 0
 
+; Set to 1 to allow using token_auths with write or admin access in iframes that embed Matomo.
+; Note that the token used will be in the URL in the iframe, and thus will be stored in webserver
+; logs and possibly other places. Using write or admin token_auths can be seen as a security risk,
+; thought it can be necessary in some use cases.
```

Comment:
is it supposed to be `though` instead of `thought`? Be good to mention specifically like `We do not recommend enabling this setting, for more information view the FAQ: ...`? with the link to the other FAQ

Reply:
👍